set.mm

```diff
@@ -547923,8 +547923,7 @@ A Fne if ( S = (/) , { X } , U. S ) ) $=
      See ~ negsym1 for more information.  (Contributed by Anthony Hart,
      11-Sep-2011.) $)
   subsym1 $p |- ( [ y / x ] [ y / x ] F. -> [ y / x ] ph ) $=
-    ( wfal wsb weq wi wa wex fal intnan nex dfsb1 mtbir pm2.21i ) DBCEZBCEZABCE
-    QBCFZPGZRPHZBIZHUASTBPRPRDGZRDHZBIZHUDUBUCBDRJKLKDBCMNKLKPBCMNO $.
+    ( wfal wsb wn nsb fal mpg pm2.21i ) DBCEZBCEZABCEKFZLFBKBCGDFMBDBCGHIIJ $.
```

Comment:
Shorter proof (no need for tags):

<pre>
subsym1 $p |- ( [ y / x ] [ y / x ] F. -> [ y / x ] ph ) $=
  ( wfal wsb sbv falim sylbi sbimi ) DBCEZABCJDADBCFAGHI $.
</pre>